applications/audio/broadcast_configuration_tool/include/structures.h

```diff
@@ -0,0 +1,269 @@
+	{.location = BT_AUDIO_LOCATION_TOP_CENTER, .name = "TC"},
+	{.location = BT_AUDIO_LOCATION_TOP_BACK_LEFT, .name = "TBL"},
+	{.location = BT_AUDIO_LOCATION_TOP_BACK_RIGHT, .name = "TBR"},
+	{.location = BT_AUDIO_LOCATION_TOP_SIDE_LEFT, .name = "TSL"},
+	{.location = BT_AUDIO_LOCATION_TOP_SIDE_RIGHT, .name = "TSR"},
+	{.location = BT_AUDIO_LOCATION_TOP_BACK_CENTER, .name = "TBC"},
+	{.location = BT_AUDIO_LOCATION_BOTTOM_FRONT_CENTER, .name = "BFC"},
+	{.location = BT_AUDIO_LOCATION_BOTTOM_FRONT_LEFT, .name = "BFL"},
+	{.location = BT_AUDIO_LOCATION_BOTTOM_FRONT_RIGHT, .name = "BFR"},
+	{.location = BT_AUDIO_LOCATION_FRONT_LEFT_WIDE, .name = "FLW"},
+	{.location = BT_AUDIO_LOCATION_FRONT_RIGHT_WIDE, .name = "FRW"},
+	{.location = BT_AUDIO_LOCATION_LEFT_SURROUND, .name = "LS"},
+	{.location = BT_AUDIO_LOCATION_RIGHT_SURROUND, .name = "RS"},
+};
+
+static inline char *location_bit_to_str(enum bt_audio_location location)
+{
+	switch (location) {
+	case BT_AUDIO_LOCATION_MONO_AUDIO:
+		return "Mono Audio";
+	case BT_AUDIO_LOCATION_FRONT_LEFT:
+		return "Front Left";
+	case BT_AUDIO_LOCATION_FRONT_RIGHT:
+		return "Front Right";
+	case BT_AUDIO_LOCATION_FRONT_CENTER:
+		return "Front Center";
+	case BT_AUDIO_LOCATION_LOW_FREQ_EFFECTS_1:
+		return "Low Frequency 1";
+	case BT_AUDIO_LOCATION_BACK_LEFT:
+		return "Back Left";
+	case BT_AUDIO_LOCATION_BACK_RIGHT:
+		return "Back Right";
+	case BT_AUDIO_LOCATION_FRONT_LEFT_OF_CENTER:
+		return "Front Left of Center";
+	case BT_AUDIO_LOCATION_FRONT_RIGHT_OF_CENTER:
+		return "Front Right of Center";
+	case BT_AUDIO_LOCATION_BACK_CENTER:
+		return "Back Center";
+	case BT_AUDIO_LOCATION_LOW_FREQ_EFFECTS_2:
+		return "Low Frequency 2";
+	case BT_AUDIO_LOCATION_SIDE_LEFT:
+		return "Side Left";
+	case BT_AUDIO_LOCATION_SIDE_RIGHT:
+		return "Side Right";
+	case BT_AUDIO_LOCATION_TOP_FRONT_LEFT:
+		return "Top Front Left";
+	case BT_AUDIO_LOCATION_TOP_FRONT_RIGHT:
+		return "Top Front Right";
+	case BT_AUDIO_LOCATION_TOP_FRONT_CENTER:
+		return "Top Front Center";
+	case BT_AUDIO_LOCATION_TOP_CENTER:
+		return "Top Center";
+	case BT_AUDIO_LOCATION_TOP_BACK_LEFT:
+		return "Top Back Left";
+	case BT_AUDIO_LOCATION_TOP_BACK_RIGHT:
+		return "Top Back Right";
+	case BT_AUDIO_LOCATION_TOP_SIDE_LEFT:
+		return "Top Side Left";
+	case BT_AUDIO_LOCATION_TOP_SIDE_RIGHT:
+		return "Top Side Right";
+	case BT_AUDIO_LOCATION_TOP_BACK_CENTER:
+		return "Top Back Center";
+	case BT_AUDIO_LOCATION_BOTTOM_FRONT_CENTER:
+		return "Bottom Front Center";
+	case BT_AUDIO_LOCATION_BOTTOM_FRONT_LEFT:
+		return "Bottom Front Left";
+	case BT_AUDIO_LOCATION_BOTTOM_FRONT_RIGHT:
+		return "Bottom Front Right";
+	case BT_AUDIO_LOCATION_FRONT_LEFT_WIDE:
+		return "Front Left Wide";
+	case BT_AUDIO_LOCATION_FRONT_RIGHT_WIDE:
+		return "Front Right Wide";
+	case BT_AUDIO_LOCATION_LEFT_SURROUND:
+		return "Left Surround";
+	case BT_AUDIO_LOCATION_RIGHT_SURROUND:
+		return "Right Surround";
+	default:
+		return "Unknown";
```

Comment:
```suggestion
		return "Unknown location";
``` 
- Or, change the string "Unknown context"; to just "Unknown"

Reply:
Will change context to "Unknown" as it is prefixed with `location: `